mindjunkies/courses/admin.py

```
@@ -18,10 +18,24 @@ class CourseAdmin(ModelAdmin):
         "course_price",
         "verified",
         "get_tags",
+        "average_rating",
+        "total_enrollments",
     )
 
     def get_tags(self, obj):
         return ", ".join(o.name for o in obj.tags.all())
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)